src/System.Private.Uri/tests/FunctionalTests/IriRelativeFileResolutionTest.cs

```diff
@@ -73,7 +73,7 @@ public void IriRelativeResolution_CompareImplcitAndExplicitFileWithUnicodeAndRes
         public void IriRelativeResolution_CompareImplcitAndExplicitUncWithNoUnicode_AllPropertiesTheSame()
         {
             string nonUnicodeImplicitTestUnc = @"\\c\path\path3\test.txt";
-            string nonUnicodeImplicitUncBase = @"\\c\path\file.txt";
+            string nonUnicodeImplicitUncBase = s_isWindowsSystem ? @"\\c\path\file.txt" : @"\\c/path/file.txt";
```

Comment:
Did this test fail due to the changes in this PR?
If so, perhaps we can reassign the _syntax for the Unc case only when !IsImplicitFile. That should exclude Uri's that ware explicitly created with a UNC string (e.g. \\server\some\path).